src/larecs/world.mojo

```diff
@@ -824,17 +762,16 @@ struct World[*component_types: ComponentType](Copyable, Movable, Sized):
         ```
 
         Args:
-            query: The query to determine which entities to remove. Note, you can
-                   either use [..query.Query] or [..query.QueryInfo].
+            filter: The filter to determine which entities to remove. Note, you can
+                   either use [..filter.MaskFilter] or [..query.Query] because [..filter.MaskFilter]
```

Comment:
Will `MaskFilter` be exposed as public API / included in the API docs? Why would that be useful? In case not, maybe simply state that a query can/should be used?